### PR TITLE
New docker hub organization: mundialis -> OSGeo

### PR DESCRIPTION
**Pull request by neteler**
The GRASS GIS docker images are currently being moved from "mundialis" to the OSGeo Docker Hub organization at https://hub.docker.com/u/osgeo

image

(and fix wrong h1 tags)

### CODE REVIEW

File: foss4g2023.html [hunk 1]
```diff
@@ -790,6 +790,16 @@ <h3>Turning git commit entries into a release page</h3>
     <p>by Vaclav Petras (procedure support by Markus Neteler)</p>
 </section>
 
+<section>
+    <h2>New docker hub organization</h2>
+    <br>
+    <h3>mundialis -> OSGeo</h3>
+    <ul>
+        <li>The GRASS GIS docker images are currently being moved from "mundialis" to the <a href="https://hub.docker.com/u/osgeo">OSGeo Docker Hub organization</a></li>
```
Comment (pesekon2) [inline foss4g2023.html]: <li>The GRASS GIS docker images are currently being moved from <a href="https://hub.docker.com/u/mundialis">mundialis</a> to the <a href="https://hub.docker.com/u/osgeo">OSGeo Docker Hub organization</a></li>
```suggestion
        <li>The GRASS GIS docker images are currently being moved from <a href="https://hub.docker.com/u/mundialis">mundialis</a> to the <a href="https://hub.docker.com/u/osgeo">OSGeo Docker Hub organization</a></li>
```

File: foss4g2023.html [hunk 2]
```diff
@@ -790,6 +790,16 @@ <h3>Turning git commit entries into a release page</h3>
...
+    </ul>
+    <img src="img/grass-dockerhub.png" alt="GRASS GIS on Docker Hub" class="stretch">
```
Comment (pesekon2) [inline foss4g2023.html]: I suggest to add (next to this screenshot) also a screenshot of <https://hub.docker.com/r/osgeo/grass-gis>. And, unfortunately, I have no idea how to add a new file to a PR as part of the review process.

**Review by pesekon2 — CHANGES_REQUESTED** [review]

File: foss4g2023.html (hunk 2 above)
Comment (neteler) [reply foss4g2023.html]: That has become very easy, using the "Code" button on the right top side of the main PRtab: you can edit the PR by checking it out like this:

image

So:
`gh pr checkout 13`

Then edit/add/commit. Then

`git push`

and the modification is  in this PR.

**Review by pesekon2 — APPROVED** [review]

**Comment by pesekon2** [comment]
@neteler: Updated with the extra screenshot. Please let me know if this looks fine for you or not.

**Comment by neteler** [comment]
> @neteler: Updated with the extra screenshot. Please let me know if this looks fine for you or not.

Great, thanks @pesekon2 !

**Pull request merged**